drivebrain_core_base/src/StateEstimator.cpp

```diff
@@ -32,6 +33,10 @@ void StateEstimator::handle_recv_process(std::shared_ptr<google::protobuf::Messa
             _vehicle_state.input = input;
             _vehicle_state.current_rpms = rpms;
         }
+    } else if (message->GetTypeName() == "hytech_msgs.VNData")
```

Comment:
We should make this a constant in case we need it in another place, and so we can easily change it if MCAP topic changes.

Reply:
while true, it will be if we need to have these in multiple places, however i think that we can do make these referenced globally / share-able when we need it to be shared